src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -274,6 +276,23 @@ tag_invoke(boost::json::value_to_tag<LedgerEntryHandler::Input>, boost::json::va
         return ripple::STXChainBridge{lockingDoor, lockingIssue, issuingDoor, issuingIssue};
     };
 
+    auto const parseOracleFromJson = [](boost::json::value const& json) -> std::optional<ripple::uint256> {
+        ripple::uint256 key = beast::zero;
+
+        if (json.is_string()) {
```

Comment:
If we put the "oracle" in indexFieldTypeMap , we don't need to have string branch here. And it can prevent the scenario that the string index is not for an actual oracle.

Reply:
Oh, interesting, thanks! 👍